src/Macros.hpp

```diff
@@ -0,0 +1,181 @@
+/*
```

Comment:
and this?

Reply:
also....is this no longer dependent on slick?

Reply:
Correct.